Routing/ClassResourceInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the FOSRestRoutingBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestRoutingBundle\Routing;
+
+use FOS\RestBundle\Routing\ClassResourceInterface as OldClassResourceInterface;
+
+if (!class_exists(OldClassResourceInterface::class)) {
+    class_alias(ClassResourceInterface::class, OldClassResourceInterface::class);
```

Comment:
What ist the reason for this?

Reply:
We want to have as less BC Breaks as possible so we still support the old used classes and namespace.

Reply:
Makes sense, but is there a reason not to do the same with the annotations for example?

Reply:
@xabbuh seems like that got lost accidently need to fix it, thank you for your review!